package.json

```diff
@@ -70,7 +70,8 @@
     "test:unit": "yarn build && mocha --config .mocharc.json './test/unit/*.test.ts' --exit",
     "test:health": "yarn build && mocha --config .mocharc.json './test/health/*.test.ts' --exit --parallel",
     "version:prepare": "yarn changeset version && yarn install --no-immutable",
-    "version:check": "yarn changeset status"
+    "version:check": "yarn changeset status",
+    "sort-yaml": "eslint --fix chains/**/*.yaml deployments/**/*.yaml"
```

Comment:
i believe this only lints when calling `yarn sort-yaml`, right?

Reply:
would love some way to run this automatically, or enforce it. Perhaps using an Action similar to what we do in monorepo https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/.github/workflows/test.yml#L80-L87